README.md

```diff
@@ -1,32 +1,36 @@
 # Dennis -- helping to release and ship it
 
-## Install
+## Setup
+
+- It's advised you use a cache directory so Dennis doesn't need to clone the repositories for every release.
+
+```
+mkdir -p ~/.dennis
+```
+
+- You may benefit from either adding a Bash alias, or from creating a one-liner script within your project, for running the releases
```

Comment:
How about adding `scripts/dennis` to the LWS and frontend repos? Less steps to explain. “Make work beautiful”.

This way you can `please dennis`, or you can add `scripts/` to your PATH.